0.10/test/test-app/README.md

```diff
@@ -2,3 +2,10 @@ node-echo
 =========
 
 node.js echo server, returns request data to response
+
+## Using existing build tools
+
+### Integrating s2i-nodejs developer support with Grunt and Gulp
+You can use Grunt or Gulp to detect changes in your local directory and sync those changes to the code running in the remote docker container.
+_Gruntfile.js_ and _gulpfile.js_ provide apt examples for doing this. 
```

Comment:
i'd rather leave this out.

1) it's not a test, so it doesn't belong in test-app imho, it's an example (you're not using it in the test script that i see)
2) it's openshift specific and requires other openshift setup details that aren't covered here.  i think it's likely to just get people into trouble.